documentation/docs/tirith-policies/tirith-policy-conditions.md

```diff
@@ -35,4 +35,25 @@ import TabItem from '@theme/TabItem';
       "item": "https://www.lambdatest.com/support/docs/tirith-policy-conditions/"
     }]
   })
-}}></script>
\ No newline at end of file
+}}></script>
```

Comment:
I think we need to add a warning sign that this policy condition documentation (and others) is incomplete and still in development. Can we do that for every `.md` that are under the `tirith-policies`?